server.js

```diff
@@ -1,33 +1,45 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import netflixData from './data/netflix-titles.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Netflix data. You can find all movies using /data, titles using /title, movies from a specific year with /year/[year] and a specific movie using /movie/[id]')
 })
 
-// Start the server
+app.get('/data', (req, res) => {
+  res.json(netflixData)
+})
+
+app.get('/title', (req, res) => {
```

Comment:
RESTful endpoints should be in plural form:) so make it /titles /years /movies, please :)